scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -24,6 +25,9 @@ workflow GvsImportGenomes {
 
   String docker_final = select_first([docker, "us.gcr.io/broad-gatk/gatk:4.1.7.0"])
 
+  # how do i return an error if the lengths are not equal?
+  # if (length(input_vcfs) != length(input_vcf_indexes) != length(research_ids))
```

Comment:
I think if you just `exit 1` from bash here, that would cause the overall task to fail